src/lib/Config.ts

```diff
@@ -58,29 +65,24 @@ class Config {
         },
       },
     };
-    const result = await prompt.get(schema);
-    Object.assign(this, result);
+    const result = await userPrompt.get(schema);
+    return new Config(result.jdkPath, result.androidSdkPath);
   }
 
-  async check() {
-    if (!await fileExists(this.jdkPath)) {
-      throw new Error(`${this.jdkPath} does not exist. Check the jdkPath on your config`);
+  static async loadConfig(): Promise<Config | null> {
```

Comment:
This is more a question - is there any reason you chose "Config | null" here instead of "Config?"? There's some info [here about it] - just curious if you had an opinion either way.

[here about it]: https://old.reddit.com/r/typescript/comments/76lk8y/interfaces_optional_property_vs_null_union_type/

Reply:
Refactored to avoid having config is a nullable.